packages/indexer-service/src/server/index.ts

```diff
@@ -288,6 +288,7 @@ export const createApp = async ({
             res
               .status(response.status || 200)
               .contentType('application/json')
+              .header('Graph-Attestable', String(response.result.attestation != null))
```

Comment:
We were not intending to surface this header as a part of the public API. It should remain an implementation detail between `graph-node` and `indexer-service` until we can merge the two and then it would go away entirely. The Consumer can check if the attestation != null to see if the response is attestable so it would also be redundant.